Assets/Mirror/Editor/Weaver/Processors/RpcProcessor.cs

```diff
@@ -82,6 +82,8 @@ public static MethodDefinition ProcessRpcCall(WeaverTypes weaverTypes, Writers w
             worker.Emit(OpCodes.Ldarg_0);
             // pass full function name to avoid ClassA.Func <-> ClassB.Func collisions
             worker.Emit(OpCodes.Ldstr, md.FullName);
+            // pass the function hash so we don't have to compute it at runtime
```

Comment:
```suggestion
            // pass the function hash so we don't have to compute it at runtime
            // otherwise each GetStableHash call requires O(N) complexity.
            // noticeable for long function names: 
            // https://github.com/MirrorNetworking/Mirror/issues/3375
```